draft-barnes-acme.md

```diff
@@ -1291,110 +1291,97 @@ The server provides the following fields as part of the challenge:
 
-serialNumbers (optional, array of numbers):
-: An array of serial numbers of certificates that are known to contain the requested key
-
-issuers (optional, array):
-: An array of X.509 Distinguished Names {{RFC5280}} of CAs that have been observed to issue certificates for this key, in text form {{RFC4514}}
-
-authorizedFor (optional, array):
-: An array of domain names, if any, for which this server regards the key as an ACME Authorized key.
 
 ~~~~~~~~~~
 
 {
   "type": "proofOfPossession",
-  "alg": "RS256",
-  "nonce": "eET5udtV7aoX8Xl8gYiZIA",
-  "hints" : {
+  "hints": {
     "jwk": {
-        "kty": "RSA",
-        "e": "AQAB",
-        "n": "KxITJ0rNlfDMAtfDr8eAw...fSSoehDFNZKQKzTZPtQ"
+      "kty": "RSA",
+      "e": "AQAB",
+      "n": "AMswMT...3aVtjE"
     },
-    "certFingerprints": [
-      "93416768eb85e33adc4277f4c9acd63e7418fcfe",
-      "16d95b7b63f1972b980b14c20291f3c0d1855d95",
-      "48b46570d9fc6358108af43ad1649484def0debf"
-    ],
-    "subjectKeyIdentifiers":  [
-      "d0083162dcc4c8a23ecb8aecbd86120e56fd24e5"
-    ],
-    "serialNumbers": [34234239832, 23993939911, 17],
-    "issuers": [
-      "C=US, O=SuperT LLC, CN=SuperTrustworthy Public CA",
-      "O=LessTrustworthy CA Inc, CN=LessTrustworthy But StillSecure"
-    ],
-    "authorizedFor": ["www.example.com", "example.net"]
+    "certs": ["MIIF7z...bYVQLY"]
   }
 }
 
 ~~~~~~~~~~
 
-In this case the server is challenging the client to prove its control over the private key that corresponds to the public key specified in the jwk object.  The signing algorithm is specified by the alg field.  The nonce value is used by the server to identify this challenge and is also used, also with a client-provided signature nonce, as part of the signature input.
+In response to this challenge, the client uses the private key corresponding to one of the acceptable public keys to sign a JWS object including data related to the challenge.  The validation object covered by the signature has the following fields:
 
-~~~~~~~~~~
+type (required, string):
+: The string "proofOfPossession"
+
+identifiers (required, identifier):
+: A list of identifiers for which the holder of the prior key authorizes the new key
 
-      signature-input = signature-nonce || server-nonce
+accountKey (required, JWK):
+: The client's account public key
 
 ~~~~~~~~~~
+{
+  "type": "proofOfPossession",
+  "identifiers: [{"type": "dns", "value": "example.com"}],
+  "accountKey": { "kty": "RSA", ... }
+}
+~~~~~~~~~~
+
+This JWS is NOT REQUIRED to have a "nonce" header parameter (as with the JWS objects that carry ACME request objects).  This allows proof-of-possession response objects to be computed off-line.  For example, as part of a domain transfer, the new domain owner might require the old domain owner to sign a proof-of-possession validation object, so that the new domain owner can present that in an ACME transaction later.
```

Comment:
I think the inconsistency of turning off nonce validation for a single challenge type is not worth the benefit for this rare situation (a domain transfer). There are a number of alternatives that do not require offline signing:
 - New domain owner or old domain owner revokes existing certificates.
 - New domain owner sends old domain owner a nonce along with the object to sign.
 - New domain owner gets a certificate from a CA that does legal existence validation, and does PoP with that certificate.